client/src/features/signup/components/UserInfoForm/UserInfoForm.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Field } from 'formik';
+import { TextField, RadioGroup } from 'formik-mui';
+import { DatePicker } from 'formik-mui-x-date-pickers';
+import { Grid, Typography, FormControlLabel, FormLabel, Radio } from '@mui/material';
+import { formModel } from '../../data';
+
+const UserInfoForm = () => {
+  const { firstName, lastName, gender, birthDate, phone } = formModel;
+
+  return (
+    <>
+      <Typography variant="h3" mb={3}>
+        User Info
+      </Typography>
+      <Grid container spacing={3} sx={{ mb: 3 }}>
+        <Grid item xs={12} laptop={6}>
```

Comment:
I guess that phone, gender and and birthday has to be required field as you have it at your props component
![image](https://user-images.githubusercontent.com/112622259/202897369-5b1fa2ed-5b16-42fa-a90b-2352a648148b.png)